fselect: add page (#2975)
tldr-pages/tldr: pages/common/fselect.md

# fselect

> > Find files with SQL-like queries.

Reviewer: ```suggestion
> Find files with SQL-like queries.
> Homepage: <https://github.com/jhspetersson/fselect>.
```

> - Find temporary or config files (full path and size):

Reviewer 2: How about - "Select full path and size from temporary or config files in a given directory."

Author: Sure, why not :)

> `fselect size, path from /home/user where name = '*.cfg' or name = '*.tmp'`

Reviewer: ```suggestion
`fselect size, path from {{path/to/directory}} where name = {{'*.cfg'}} or name = {{'*.tmp'}}`
```

- Find square images:

> `fselect path from /home/user/Photos where width = height`

Reviewer: ```suggestion
`fselect path from {{path/to/directory}} where width = height`
```

> - Find old-school rap MP3 files:

Reviewer: ```suggestion
- Find old-school rap 320kbps MP3 files:
```

> `fselect path from /home/user/music where genre = Rap and bitrate = 320 and mp3_year lt 2000`

Reviewer: ```suggestion
`fselect path from {{path/to/directory}} where genre = {{Rap}} and bitrate = {{320}} and mp3_year lt {{2000}}`
```

> - Output as json:

Reviewer: ```suggestion
- Select only the first 5 results and output as JSON:
```

> `fselect size, path from /home/user limit 5 into json`

Reviewer: ```suggestion
`fselect size, path from {{path/to/directory}} limit {{5}} into json`
```

> - Aggregate functions:

Reviewer: ```suggestion
- Use SQL aggregate functions to calculate minimum, maximum and average size of files in a directory:
```

> `fselect "MIN(size), MAX(size), AVG(size), SUM(size), COUNT(*) from /home/user/Downloads"`

Reviewer: ```suggestion
`fselect "{{MIN(size), MAX(size), AVG(size), SUM(size), COUNT(*)}} from {{path/to/directory}}"`
```

# fselect

> Find files with SQL-like queries.

> > Homepage: <https://github.com/jhspetersson/fselect>.

Reviewer: ```suggestion
> More information: <https://github.com/jhspetersson/fselect>.
```
I missed this, sorry. Looks good after this.
